tests/functional/stax/navigator.py

```diff
@@ -0,0 +1,79 @@
+from enum import auto, Enum
+from functools import partial
+from ragger.navigator.navigator import Navigator
+from time import sleep
+
+from .screen import StaxScreen
+
+
+class NavInsID(Enum):
+    WAIT = auto()
+    TOUCH = auto()
+    # home screen
+    HOME_TO_QUIT = auto()
```

Comment:
I think you should make sure there is no enum ID reuse from ragger ones, else I fear some conflicts.

Reply:
As this class replaces (and does not extend) the default ragger `NavInsID` enum, there won't be any conflict.

Although I agree that it would be better if we could extend the enum - less repetition and straighter code. It's both a limitation of the Python builtin `enum` type, and of `Ragger`.

Reply:
I think there might be some conflict here: https://github.com/LedgerHQ/ragger/blob/develop/src/ragger/navigator/navigator.py#L209. This function as specific behavior depending on the ID, and yours might be the same...

Reply:
The ID value is not an raw type (int for instance), but will be of type `<enum 'NavInsID'>`, and will look something like `<NavInsID.USE_CASE_REVIEW_CONFIRM: 17>`, so even if the final value (`NavInsID.USE_CASE_REVIEW_CONFIRM.value`) is the same, there should not be any colision.